<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="11" passed="8" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-08-02T15:38:34 UTC" name="PetSuite" finished-at="2024-08-02T15:38:51 UTC" duration-ms="16702">
    <groups>
    </groups>
    <test started-at="2024-08-02T15:38:34 UTC" name="PetTest" finished-at="2024-08-02T15:38:51 UTC" duration-ms="16702">
      <class name="api.test.UserModelTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:api.test.UserModelTest@52bf72b5]" started-at="2024-08-02T15:38:45 UTC" name="setup" finished-at="2024-08-02T15:38:46 UTC" duration-ms="231" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testCreateUser()[pri:1, instance:api.test.UserModelTest@52bf72b5]" started-at="2024-08-02T15:38:46 UTC" name="testCreateUser" finished-at="2024-08-02T15:38:47 UTC" duration-ms="1029" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateUser -->
        <test-method signature="testGetUserName()[pri:1, instance:api.test.UserModelTest@52bf72b5]" started-at="2024-08-02T15:38:47 UTC" name="testGetUserName" finished-at="2024-08-02T15:38:48 UTC" duration-ms="990" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetUserName -->
        <test-method signature="testDeleteUserByName(java.lang.String)[pri:2, instance:api.test.UserModelTest@52bf72b5]" started-at="2024-08-02T15:38:48 UTC" name="testDeleteUserByName" data-provider="UserNames" finished-at="2024-08-02T15:38:49 UTC" duration-ms="986" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at api.test.UserModelTest.testDeleteUserByName(UserModelTest.java:71)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUserByName -->
        <test-method signature="testDeleteUserByName(java.lang.String)[pri:2, instance:api.test.UserModelTest@52bf72b5]" started-at="2024-08-02T15:38:49 UTC" name="testDeleteUserByName" data-provider="UserNames" finished-at="2024-08-02T15:38:50 UTC" duration-ms="951" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser2]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at api.test.UserModelTest.testDeleteUserByName(UserModelTest.java:71)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUserByName -->
        <test-method signature="testDeleteUserByName(java.lang.String)[pri:2, instance:api.test.UserModelTest@52bf72b5]" started-at="2024-08-02T15:38:50 UTC" name="testDeleteUserByName" data-provider="UserNames" finished-at="2024-08-02T15:38:51 UTC" duration-ms="954" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser3]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at api.test.UserModelTest.testDeleteUserByName(UserModelTest.java:71)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUserByName -->
      </class> <!-- api.test.UserModelTest -->
      <class name="api.test.UserDDTest">
        <test-method signature="testPostuser(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:api.test.UserDDTest@795cd85e]" started-at="2024-08-02T15:38:37 UTC" name="testPostuser" data-provider="Data" finished-at="2024-08-02T15:38:40 UTC" duration-ms="3336" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[1010]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[testuser1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[John]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Canedy]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[a@a.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[test@1]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[123456789]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPostuser -->
        <test-method signature="testPostuser(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:api.test.UserDDTest@795cd85e]" started-at="2024-08-02T15:38:40 UTC" name="testPostuser" data-provider="Data" finished-at="2024-08-02T15:38:41 UTC" duration-ms="1006" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[1020]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[testuser2]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[kim]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Kom]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[b@a.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[test@2]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[123456789]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPostuser -->
        <test-method signature="testPostuser(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:api.test.UserDDTest@795cd85e]" started-at="2024-08-02T15:38:41 UTC" name="testPostuser" data-provider="Data" finished-at="2024-08-02T15:38:42 UTC" duration-ms="961" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[1030]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[testuser3]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Smith]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Steve]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[c@a.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[test@3]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[123456789]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPostuser -->
        <test-method signature="testDeleteUserByName(java.lang.String)[pri:2, instance:api.test.UserDDTest@795cd85e]" started-at="2024-08-02T15:38:42 UTC" name="testDeleteUserByName" data-provider="UserNames" finished-at="2024-08-02T15:38:43 UTC" duration-ms="1010" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser1]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUserByName -->
        <test-method signature="testDeleteUserByName(java.lang.String)[pri:2, instance:api.test.UserDDTest@795cd85e]" started-at="2024-08-02T15:38:43 UTC" name="testDeleteUserByName" data-provider="UserNames" finished-at="2024-08-02T15:38:44 UTC" duration-ms="961" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser2]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUserByName -->
        <test-method signature="testDeleteUserByName(java.lang.String)[pri:2, instance:api.test.UserDDTest@795cd85e]" started-at="2024-08-02T15:38:44 UTC" name="testDeleteUserByName" data-provider="UserNames" finished-at="2024-08-02T15:38:45 UTC" duration-ms="976" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUserByName -->
      </class> <!-- api.test.UserDDTest -->
    </test> <!-- PetTest -->
  </suite> <!-- PetSuite -->
</testng-results>
